Library/Homebrew/utils/github/actions.rb

```diff
@@ -56,8 +67,22 @@ def to_s
 
           if @line
             metadata << ",line=#{@line}"
-            metadata << ",col=#{@column}" if @column
+            metadata << ",endLine=#{@end_line}" if @end_line
+
+            if @column
+              metadata << ",col=#{@column}"
+              metadata << ",endColumn=#{@end_column}" if @end_column
+            end
+          end
+        end
+
+        if @title
+          metadata << if @file
+            ","
+          else
+            " "
           end
+          metadata << "title=#{Actions.escape(@title)}"
```

Comment:
Could maybe adjust the existing `brew test failed!` message to be in the title instead and have a `Click for output` for the `<summary`?